dataset/src/main/scala/frameless/ops/ColumnTypes.scala

```diff
@@ -26,3 +27,22 @@ object ColumnTypes {
   ): ColumnTypes.Aux[T, TypedColumn[T, H] :: TT, H :: V] =
     new ColumnTypes[T, TypedColumn[T, H] :: TT] {type Out = H :: V}
 }
+
+
+trait ColumnFieldTypes[T, U <: HList] {
```

Comment:
Same here ... why do we need these?

Reply:
I'm pretty sure this is now unneeded; the point was to compute `FieldType[KA, A] :: FieldType[KB, B] :: HNil` from `TypedColumn[T, FieldType[KA, A]] :: TypedColumn[T, FieldType[KB, B]] :: HNil`, but this should be covered by `ColumnTypes`.

Added during my tries to make `selectManyRecord` work :-)